concourse/tasks/daisy-build-images-windows-container.yaml

```diff
@@ -0,0 +1,25 @@
+platform: linux
```

Comment:
what is different about this task compared to the other daisy-build-images tasks? it looks copied from them

Reply:
It's indeed almost an exact copy. We are using a different set of variables with container task. Are there ways in Concourse task allow us to define which set of variables to pass to the downstream `run`?

Reply:
yes, several ways. if the container uses a shell, the easiest thing to do is set and use environment variables in the task, allowing them to be overridden in the pipeline invocation. if not, using interpolation for required variables

what differs in this task vs the existing?

Reply:
The differences are in the variables and the variable values. I am a bit unsure if in this case we can gain simplicity  switching to using the environment variable and overriding. My understanding of Concourse is still pretty early on and may be unaware of other possibilities. 

|daisy-build-images-windows-container|`daisy-build-images.yaml`|
|---|---|
| - -project=gce-image-builder<br>- -zone=us-central1-c<br>- -var:build_date=((build_date))<br>- -var:source_image_project:((source_image_project))<br>- -compute_endpoint_override=https://www.googleapis.com/compute/beta/projects/<br>- ./compute-image-tools/daisy_workflows/build-publish/((wf))| - -project=gce-image-builder<br>- -zone=us-central1-c<br>- -var:installer_iso=((iso))<br>- -var:build_date=((build_date))<br>- -var:google_cloud_repo=((google_cloud_repo))<br>- -var:gcs_url=((gcs_url))<br>- -compute_endpoint_override=https://www.googleapis.com/compute/beta/projects/<br>- ./compute-image-tools/daisy_workflows/build-publish/((wf))|